docker-compose-tests.yml

```diff
@@ -36,7 +36,7 @@ services:
       - gp-finder-db:mongo
 
   gp-finder-db:
-    image: "nhsuk/profiles-db:${profiles_db_docker_image_tag-0.3.0}"
+    image: "nhsuk/profiles-db:${profiles_db_docker_image_tag-0.4.0}"
```

Comment:
This version has the pomi data. There are tests coupled to the data too - for which we have to come up with a solution with, but this is what we have at the moment.